### DISCUSSION TITLE
Custom "From" any domain name

### DISCUSSION THREAD

**Discussion by charlesmudy** [postalserver/postal#1609] [Help with using Postal] [answered]
Hello,

I'm trying to understand how I can use any custom domain name in the "From" field which says only validated domain name.

Sendgrid and other email service does this, how can I achieve the same?

**Comment by miroslavskela** [comment, marked as answer]
You can enable that from portal. Choose Mail Server for which one you want this to work, then settings then advanced settings, and ALLOW SENDER HEADER change to Yes. Keep in mind that you have to add sender header when sending from custom "from".

**Reply by charlesmudy** [reply]
Amazing @miroslavskela  ... this work via the API and not directly from the Postal dashboard. Not sure where to set the 'Sender' header.

Thanks anyways.

**Reply by willpower232** [reply]
If you're using the API then you can supply the sender as a separate parameter https://apiv1.postalserver.io/controllers/send/message.html
